tox.ini

```diff
@@ -22,6 +23,9 @@ commands =
     flake8 reconcile tools release e2e_tests dockerfiles/hack setup.py
     pylint --extension-pkg-whitelist='pydantic' reconcile tools release e2e_tests dockerfiles/hack setup.py
 
+[testenv:imports]
```

Comment:
Can you `skip_install` here or would it break something? We do it on `format` and it saves a ton of time if it's possible.

Reply:
I've added `skip_install`. Thanks for the hint.